st2common/st2common/models/system/actionchain.py

```diff
@@ -86,6 +92,24 @@ def __init__(self, **kw):
             prop = string.replace(prop, '-', '_')
             setattr(self, prop, value)
 
+    def validate(self):
```

Comment:
I just noticed we don't actually validate Node parameters against the schema defined above (or am I missing something and we do validate it somewhere?).

If we indeed don't, we should also update this method to validate it here - I'm kind reluctant to do it in this PR thought since who knows what kind of issues it might expose.